ieeg_visual_multimodal/task-temporalpattern_events.json

```diff
@@ -0,0 +1,41 @@
+{
+    "onset": {
+        "Description": "Onset of the event",
+        "Units": "seconds"
+    },
+    "duration": {
+        "Description": "Duration of the event",
+        "Units": "seconds"
+    },
+    "ISI": {
+        "Description": "Interval between two-pulse stimuli. Applies only to TWO-PULSE condition",
+        "Units": "seconds"
+    },
+    "trial_type": {
```

Comment:
are these trigger codes ("TTL" event markers) that are present in the data? If so, I would suggest renaming this to `value` as described in the table in https://bids-specification.readthedocs.io/en/latest/04-modality-specific-files/05-task-events.html

Also: "integers" is not really a unit, so I would leave that out and make the point in the "Description"

Reply:
They are not trigger codes, we have only a binary trigger code (an audio pulse) in our ECoG data so these events were identified after data collection by aligning them with the binary triggers (the trigger channel is included in the data).